Documentation/dev-guide/apispec/swagger/rpc.swagger.json

```diff
@@ -2771,6 +2771,10 @@
           "type": "string",
           "format": "uint64"
         },
+        "storageVersion": {
+          "description": "storageVersion is the version of the db file.",
```

Comment:
I would add something like: 
  " It might get updated with delay in relationship to the binary version powering the etcd-instance and target version of the cluster."

Reply:
Makes sense. 

The etcd binary version is hard coded in the [source code](https://github.com/etcd-io/etcd/blob/main/api/version/version.go#L29), so it will never be updated in runtime environment, unless users manually replace the binary. 

So we don't need to mention the binary version here. I will change the description to: 
`storageVersion is the version of the db file. It might be get updated with delay in relationship to the target cluster version.`

Reply:
Done & thanks.